compiler/one-cmds/one-import-tf

```diff
@@ -14,199 +18,175 @@
-  echo "Error: input model not found"
-  echo ""
-  usage
-  exit 2
-fi
-
-if [ -z ${INPUT_ARRAYS} ]; then
-  input_arrays_not_set
-fi
-
-# INPUT_SHAPES is optional
-
-if [ -z ${OUTPUT_PATH} ]; then
-  output_not_set
-fi
-
-if [ -z ${OUTPUT_ARRAYS} ]; then
-  output_arrays_not_set
-fi
-
-FILE_BASE=$(basename ${OUTPUT_PATH})
-MODEL_NAME="${FILE_BASE%.*}"
-
-TMPDIR=$(mktemp -d)
-trap "{ rm -rf $TMPDIR; }" EXIT
-
-# activate python virtual environment
-VIRTUALENV_LINUX="${DRIVER_PATH}/venv/bin/activate"
-VIRTUALENV_WINDOWS="${DRIVER_PATH}/venv/Scripts/activate"
-
-if [ -e ${VIRTUALENV_LINUX} ]; then
-  source ${VIRTUALENV_LINUX}
-elif [ -e ${VIRTUALENV_WINDOWS} ]; then
-  source ${VIRTUALENV_WINDOWS}
-else
-  echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command."
-  echo "If encounter any problems, please follow provided document in 'doc' folder."
-  exit 255
-fi
-
-# remove previous log
-rm -rf "${OUTPUT_PATH}.log"
-
-show_err_onexit()
-{
-  cat "${OUTPUT_PATH}.log"
-}
-
-trap show_err_onexit ERR
-
-# generate temporary tflite file
-CONVERT_SCRIPT="python ${DRIVER_PATH}/tf2tfliteV2.py ${TF_INTERFACE} "
-CONVERT_SCRIPT+="--input_path ${INPUT_PATH} "
-CONVERT_SCRIPT+="--input_arrays ${INPUT_ARRAYS} "
-CONVERT_SCRIPT+="--output_path ${TMPDIR}/${MODEL_NAME}.tflite "
-CONVERT_SCRIPT+="--output_arrays ${OUTPUT_ARRAYS} "
-if [ ! -z ${INPUT_SHAPES} ]; then
-  CONVERT_SCRIPT+="--input_shapes ${INPUT_SHAPES} "
-fi
-
-echo ${CONVERT_SCRIPT} > "${OUTPUT_PATH}.log"
-echo "" >> "${OUTPUT_PATH}.log"
-$CONVERT_SCRIPT >> "${OUTPUT_PATH}.log" 2>&1
-
-# convert .tflite to .circle
-echo " " >> "${OUTPUT_PATH}.log"
-echo "${DRIVER_PATH}/tflite2circle" "${TMPDIR}/${MODEL_NAME}.tflite" \
-"${OUTPUT_PATH}" >> "${OUTPUT_PATH}.log"
-echo " " >> "${OUTPUT_PATH}.log"
-
-"${DRIVER_PATH}/tflite2circle" "${TMPDIR}/${MODEL_NAME}.tflite" \
-"${OUTPUT_PATH}" >> "${OUTPUT_PATH}.log" 2>&1
+import argparse
+import os
+import subprocess
+import sys
+import tempfile
+
+import utils as _utils
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to convert TensorFlow to circle')
+
+    _utils._add_default_arg(parser)
+
+    ## tf2tfliteV2 arguments
+    tf2tfliteV2_group = parser.add_argument_group('converter arguments')
+
+    # converter version
+    converter_version = tf2tfliteV2_group.add_mutually_exclusive_group()
+    converter_version.add_argument(
+        '--v1',
+        action='store_const',
+        dest='converter_version',
+        const='--v1',
+        help='use TensorFlow Lite Converter 1.x')
+    converter_version.add_argument(
+        '--v2',
+        action='store_const',
+        dest='converter_version',
+        const='--v2',
+        help='use TensorFlow Lite Converter 2.x')
+
+    converter_version.set_defaults(converter_version='--v1')
+
+    # input model format
+    model_format_arg = tf2tfliteV2_group.add_mutually_exclusive_group()
```

Comment:
Supporting saved model and keras model is new feature for `one-import-tf`
Please prepare test models and test scripts for these: you may add them in another PR